SingularityService/src/main/java/com/hubspot/singularity/hooks/LoadBalancerClientImpl.java

```diff
@@ -200,4 +206,15 @@ public SingularityLoadBalancerUpdate cancel(LoadBalancerRequestId loadBalancerRe
 
     return sendRequestWrapper(loadBalancerRequestId, LoadBalancerMethod.CANCEL, requestBuilder.build(), BaragonRequestState.UNKNOWN);
   }
+
+  @Override
+  public SingularityLoadBalancerUpdate delete(LoadBalancerRequestId loadBalancerRequestId, SingularityRequest request, SingularityDeploy deploy) {
+    final List<String> serviceOwners = request.getOwners().or(Collections.<String> emptyList());
+    final Set<String> loadBalancerGroups = deploy.getLoadBalancerGroups().or(Collections.<String> emptySet());
+    final BaragonService lbService = new BaragonService(request.getId(), serviceOwners, deploy.getServiceBasePath().get(), loadBalancerGroups, deploy.getLoadBalancerOptions().orNull());
```

Comment:
Does the service not take optionals ? Seems awkward to have to transition between Optionals and empty collections here.